Fix playoff series display in new API

parnic opened the pull request:
This playoff endpoint data was pulled from https://github.com/Zmalski/NHL-API-Reference?tab=readme-ov-file#playoff-information

parnic committed:
Fix playoff series display in new API

This playoff endpoint data was pulled from https://github.com/Zmalski/NHL-API-Reference?tab=readme-ov-file#playoff-information

parnic commented:
This gets the display from having no Series info to, e.g.:

![image](https://github.com/fewieden/MMM-NHL/assets/888126/cd280c28-ecd9-4f3e-b302-48713aee5a2c)

parnic committed:
Satisfy PR checks

Updates changelog

I didn't make any changes to the functions that the linter was complaining about, so I just made some token comments to make it happy.

parnic committed:
Remove trailing spaces

parnic committed:
Fix round filter

I was comparing against the wrong variable as well as not correctly handling the transition from one round to the next.

caseybecking commented:
I have this PR running and it is working as expected with the playoffs.

dannoh commented:
Looks good, I think we need to rev the version in the package.json though @parnic 

@fewieden

parnic committed:
Increase version

parnic commented:
> Looks good, I think we need to rev the version in the package.json though @parnic
> 
> @fewieden


Sure thing. Done.

parnic committed:
Compute which round(s) to show ourselves

The new NHL API returns the current round as, for example, 3 even when all series from round 2 haven't been completed. This adds detection of series which haven't been played all the way out and sets that as the current round.

parnic commented:
I am maintaining https://github.com/parnic/MMM-NHL for now until @fewieden makes his return :). I will be keeping that master branch updated for this season.

KristjanESPERANTO commented:
> I am maintaining https://github.com/parnic/MMM-NHL for now until @fewieden makes his return :). I will be keeping that master branch updated for this season.

Nice! Thank you! :rocket: 

I have changed the URL for the module to your fork in [the official module list](https://github.com/MagicMirrorOrg/MagicMirror/wiki/3rd-Party-Modules) so that other users can benefit from your changes. As soon as you no longer maintain your fork, we can of course change it back again.

Please activate the issues tab in your fork.

parnic commented:
> Please activate the issues tab in your fork.

Done.
